src/main/java/com/revature/Flumblr/dtos/requests/NewPostRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.revature.Flumblr.dtos.requests;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class NewPostRequest {
+    private String id;
+    
+    private String username;
+
+    private String message;
+
+    private String s3Url;
+
+    private String mediaType;
+
+    private String createTime;
```

Comment:
no need for createTime and editTime in requests. They are generated in constructor of entities

Reply:
also need array of tags
so String[] tags